CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 ## [1.4.0] - 2019-05-23
 
 ### Added
+- Option in UVBase to check acceptability of set parameters, ignoring whether all required parameters are set.
```

Comment:
Again, I think this is confused between the acceptability check that the basic check.